visualization/src/pcl_visualizer.cpp

```diff
@@ -3979,7 +3985,7 @@ pcl::visualization::PCLVisualizer::renderViewTesselatedSphere (
       triangle->GetPoints ()->GetPoint (1, p1);
       triangle->GetPoints ()->GetPoint (2, p2);
       visible_area += vtkTriangle::TriangleArea (p0, p1, p2);
-    }
+  }
```

Comment:
This also looks like a mistake. The indentation was similar to the opening bracket, before the change.